app/presenters/document/web_resource_presenter.rb

```diff
@@ -0,0 +1,175 @@
+module Document
+  ##
+  # Blacklight document presenter for a Europeana web resource
+  class WebResourcePresenter < DocumentPresenter
+    include ActionView::Helpers::NumberHelper
+
+    def initialize(document, record, controller, configuration = controller.blacklight_config)
+      super(document, controller, configuration)
+      @record = record
+      @record_presenter = RecordPresenter.new(record, controller)
+    end
+
+    def media_item
```

Comment:
Assignment Branch Condition size for media_item is too high. [16.03/15]<br>Method has too many lines. [19/10]